tests/rptest/test_suite_func.yml

```diff
@@ -0,0 +1,15 @@
+# TODO work in progress getting clustered mode issues worked out
```

Comment:
test_suite_func is a strange name?  Should this be called test_suite_gcp_temporary or something like that to make it obvious what it is, and that it will go away?

Reply:
My intention was to create a core functional test suite which can be executed on any form factor or cloud provider, excluding the scale tests. You are right, I should have had a better name conveying the purpose.

Reply:
Renamed the suite file accordingly. https://github.com/redpanda-data/redpanda/pull/9033/commits/182a9b6bb839aa0b27c7c8862272f2786bd8981e